### DISCUSSION TITLE
Select `defaultValue` ignored

### DISCUSSION THREAD

**Discussion by jalexy12** [twilio-labs/paste#3521] [General] [answered]
Hey all,

We were migrating a few components over to Paste and realized the `Select` seems to be ignoring the `defaultValue` prop. 

It actually doesn't work on the demo page either, as seen here: https://paste.twilio.design/components/select#select-with-default-value

(The default selected should be dogs, but is actually cats)

Thanks!

**Comment by SiTaggart** [comment]
Hi @jalexy12 

That definitely seems like a bug. I'll try and knock something out this week.

Thanks for the report.

**Reply by jalexy12** [reply]
Thanks!

**Reply by serifluous** [reply, marked as answer]
You can follow along the fix in PR #3545

**Reply by SiTaggart** [reply]
As an FYI this went out in Core `v20.5`
